aten/src/ATen/native/cpu/group_norm_kernel.cpp

```diff
@@ -166,7 +165,7 @@ void GroupNormKernelImplChannelsLastInternal(
     int64_t C,
     int64_t HxW,
     int64_t group,
-    T eps,
+    double eps,
```

Comment:
why eps needs double here? What about float?

Reply:
revert this to align with the cuda kernels https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cuda/group_norm_kernel.cu#L554